### DISCUSSION TITLE
Normalized effect size to tell the direction of enrichment

### DISCUSSION THREAD

**Discussion by eclarsen-10** [egeulgen/pathfindR#156] [Q&A]
Hi,

I am just wondering if there is currently anything implemented in the package and/or pathfindR functions that will output the normalized effect size, or an effect size, such that we can tell if the enrichments show that specific terms are activated or suppressed?

This is kind of like what can be done with clusterProfiler. To generate graphs like attached
![example](https://user-images.githubusercontent.com/72929581/217913291-aee127f5-4df6-49d0-b358-a5bebc7ff76b.JPG)

**Comment by egeulgen** [comment]
hello, thank you for your question. While we don't currently have such functionality but I will try to implement this in an upcoming version and update you
